plat/Config.uk

```diff
@@ -39,7 +39,7 @@ config UKPLAT_LCPU_MAXCOUNT
 config HAVE_SMP
```

Comment:
Commit `plat/common: Adapt common code to the new uk_intctlr API` has no commit message, just something short would suffice, but let's try to not leave it empty. I assume you will remove `Notice: Picking individual commits in this PR will break the build.` before merge, right?

Reply:
@skuenzer wanted to have these when individual commits in a PR break the build for folks who do `git bisect` 🤷🏼‍♂️

Reply:
Ah, I see, actually a good idea :+1:, especially since commits end up having that `Closes: #<PR>` tag at the end added automatically.